docs/source/api_reference/forecasting.rst

```diff
@@ -350,7 +350,7 @@ Structural time series models
 
     DynamicFactor
 
-Transformer (deep learning) based forecasters
+Deep learning based forecasters
```

Comment:
May I request to not change this? If `neuralforecast` PR (#5962) is approved and merged, it'll have conflicts, but more importantly we'll be able to add more deep learning based forecasters and it may be better to group them into transformer based, RNN based, etc.

Reply:
I think you need not to revert the changes. Since If I do understand it correctly, the mentioned LTSF forecaster are not transformers.